frontend/src/components/Layout/index.tsx

```diff
@@ -9,8 +10,9 @@ import { Wrapper } from "./style";
 const Layout: FC = ({ children }) => {
   const { pathname } = useRouter();
 
-  // CHANGE TO "/" FOR PROD
-  if (pathname === "/landing-page") {
+  console.log(pathname);
```

Comment:
Do we want to keep the `console.log` here?

Reply:
TY!

Reply:
```suggestion
```